### DISCUSSION TITLE
ERROR: Unsupported distribution 'rocky'

### DISCUSSION THREAD

**Discussion by ghost** [btcpayserver/btcpayserver#6059] [Technical Support]
I tried installing btcpay server on "rocky linux" but i get the following message:

ERROR: Unsupported distribution 'rocky'

Does this mean it won't work on Rocky Linux?  

I am going to assume it only works on ubuntu? 

Why won't it work with rocky linux

**Comment by mp21-org** [comment]
replied to you on [Mattermost](https://chat.btcpayserver.org/btcpayserver/pl/1pu11er1ffywdg8beq5cw6qsqr).
